nucliadb/nucliadb/ingest/orm/knowledgebox.py

```diff
@@ -486,17 +486,22 @@ async def purge(cls, driver: Driver, kbid: str):
         await cls.delete_all_kb_keys(driver, kbid)
 
     @classmethod
-    async def delete_all_kb_keys(cls, driver: Driver, kbid: str):
+    async def delete_all_kb_keys(cls, driver: Driver, kbid: str, chunk_size=1_000):
```

Comment:
I think chunk size should not be a parameter, as it's a detail not important for callers wanting to delete all KB keys

Reply:
agree! Changed in my last commit